app/services/README.md

```diff
@@ -5,8 +5,8 @@ a single place that instantiates all the services related to a model
 
 ## Conventions
 
-- Use the `execute` method as the interface on the service
-- Use named arguments to call `execute` to explicitly state what the arguments going in are
+- Use the `call` method as the interface on the service
+- Use named arguments to call `call` to explicitly state what the arguments going in are
```

Comment:
Is 'keyword arguments' the correct term here?

Reply:
I think it depends on the service, I'm not sure we're dictating whether they are keyword arguments are regular arguments here.